src/io/sampling.jl

```diff
@@ -193,10 +193,10 @@ function sample(
 end
 
 """
-    sample_site_selection(d::Domain, n::Int64, sample_method=SobolSample(R=OwenScramble(base=2, pad=32)))::DataFrame
+    sample_selection(d::Domain, n::Int64, sample_method=SobolSample(R=OwenScramble(base=2, pad=32)))::DataFrame
```

Comment:
Maybe `sample_location_selection`? `sample_selection` doesn't immediately remind me what the function does